packages/perseus-core/src/parse-perseus-json/perseus-parsers/perseus-answer-area.test.ts

```diff
@@ -0,0 +1,53 @@
+import {ctx} from "../general-purpose-parsers/test-helpers";
+import {success} from "../result";
+
+import {parsePerseusAnswerArea} from "./perseus-answer-area";
+
+import type {PerseusAnswerArea} from "@khanacademy/perseus-core";
+
+describe("parsePerseusAnswerArea", () => {
+    const allFalse: PerseusAnswerArea = {
+        zTable: false,
+        calculator: false,
+        chi2Table: false,
+        financialCalculatorMonthlyPayment: false,
+        financialCalculatorTotalAmount: false,
+        financialCalculatorTimeToPayOff: false,
+        periodicTable: false,
+        periodicTableWithKey: false,
+        tTable: false,
+    };
+
+    it("fills in missing fields with `false`", () => {
```

Comment:
No action needed, but noting that we don't test a mix between false/true values:

```
const result = parsePerseusAnswerArea({
  calculator: true,
  chi2Table: false,
}, ctx());
```

Reply:
We're also not testing that each input field is mapped to the correct output field. As I've written the test, there's nothing stopping us from scrambling the fields when we assign them.

I'm inclined not to test for such things; tests provide value when they catch our mistakes, so a test for a mistake we're unlikely to ever make is unlikely to provide value.